MANIFEST.in

```diff
@@ -16,6 +16,7 @@ recursive-include tools *.py *.sh
 recursive-include mdn *.py
 recursive-include mdn/jinja2 *.html
 recursive-include mdn/static *.css
+recursive-include requirements *.txt
```

Comment:
Do we also need to pick up `requirements-rtd.txt`? Should that file be in `requirements`, too?